package.json

```diff
@@ -688,7 +688,7 @@
     "scripts": {
         "vscode:prepublish": "yarn compile-production",
         "compile": "yarn install && tsc -p ./",
-        "compile-production": "yarn install && yarn run translations-generate && yarn run compile",
+        "compile-production": "yarn install && yarn run translations-generate",
```

Comment:
Removing the ending "&& yarn run compile" from the "compile-production" script, will not result in not compiling the project? I don't see it added somewhere else.

All else looks good and thank you very much for helping me figure out the whole localization functionality. I will test right now.

Reply:
Never mind... I just read your PR description again where you explain why you remove "yarn run compile".
The vsix I built from your branch doesn't work for me but maybe there's something else going on from my side, I'll figure out and let you know.

Reply:
I think it was a problem of installation on top of what was there. I uninstalled the Makefile extension and reapplied the vsix from your branch and all works great now, all strings. Thank you very much Colen. I'll merge this in.